components/entity/src/main/resources/entityengine.xml

```diff
@@ -24,7 +24,7 @@ access. For a detailed description see the core/docs/entityconfig.html file.
 <entity-config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="https://ofbiz.apache.org/dtds/entity-config.xsd">
     <resource-loader name="fieldfile" class="org.apache.ofbiz.base.config.FileLoader"
-            prepend-env="ofbiz.home" prefix="/framework/entity/fieldtype/"/>
+            prepend-env="user.dir" prefix="/components/entity/src/main/resources/fieldtype/"/>
```

Comment:
Entity loading should be done by the app, not by the library.
Right now entity loading is started during class initialization.
IMO this should be done by a call in the app that says: "I am starting, I will configure all the things I want, how I want" . 

This way of doing things is used across the codebase (UtilProperties, Cache) via static initialization code. 
This works of for application but for library, it should be separated:
- library provides the means of doing things = code to load and parse entities
- application does those things = load the entity definition files and parse them